site/content/contribute/mobile/push-notifications/_index.md

```diff
@@ -0,0 +1,16 @@
+---
+title: "Setup Push Notifications"
+date: 2015-05-20T11:35:32-04:00
+weight: 6
+subsection: Mobile Apps
+---
+
+When building a custom version of the Mattermost mobile app, you will also need to host your own [Mattermost Push Notification Service](https://github.com/mattermost/mattermost-push-proxy/releases) and make a few modifications to your Mattermost mobile app to be able to get push notifications.
+
+1. Setup the custom mobile apps to receive push notifications
+    - [Android](/contribute/mobile/push-notifications/android)
+    - [iOS](/contribute/mobile/push-notifications/ios)
+2. [Setup the Mattermost Push Notification Service](/contribute/mobile/push-notifications/service)
+
+
+{{% note "Migration" %}} As of April 10, 2018, Google has deprecated the [Google Cloud Messaging (GCM) service](https://developers.google.com/cloud-messaging/gcm). The GCM server and client APIs are deprecated and will be removed as of April 11, 2019. **If you are running the Mattermost Push Notification Service v5.4 or earlier**, you must [migrate to Firebase Cloud Messaging (FCM)](/contribute/mobile/push-notificaions/migrate-gcm-fcm){{% /note %}}                                                                                                                                                                                                                                                                                                                                                                                                                                                                          
```

Comment:
The note formatting is not showing on preview

Reply:
This is a hugo feature

![image](https://user-images.githubusercontent.com/6757047/54537197-e2198980-4970-11e9-9d99-94c63384c821.png)